### PR TITLE
Add convenience integration test running script

### PR DESCRIPTION
**Pull request by kenxu95**
## Describe your changes and why you are making these changes
Similar to how `run_linters.py` made our lives easier, `run_tests.py` does the same thing for our SDK Integration Tests in the test directory.

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: manual_qa_tests/workflows/succeed_parameters.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import aqueduct as aq
 import time
 
+import aqueduct as aq
```
Comment (kenxu95) [inline manual_qa_tests/workflows/succeed_parameters.py:3]: weird, I'm pretty sure my python linter up to date?

File: integration_tests/sdk/README.md [hunk 2]
```diff
@@ -31,21 +31,23 @@ see `aqueduct_tests/conftest.py` and  `data_integration_tests/conftest.py`.
...
+To run all SDK Integration Tests:
+`python3 run_tests.py [-lf] [-n CONCURRENCY]`
 
-To run the Data Integration Tests:
-`pytest data_integration_tests/ -rP -vv`
+To run just one of the test suites:
+- `python3 run_tests.py --aqueduct [-lf] [-n CONCURRENCY]`
+- `python3 run_tests.py --data-integration [-lf] [-n CONCURRENCY]`
 
+The run tests with concurrency > 1, `pytest-xdist` must be installed.
```
Comment (saurav-c) [inline integration_tests/sdk/README.md:41]: nit: To run tests...

File: integration_tests/sdk/README.md [hunk 3]
```diff
@@ -31,21 +31,23 @@ see `aqueduct_tests/conftest.py` and  `data_integration_tests/conftest.py`.
 
 ## Commands
 
-From this directory, to run the Aqueduct Tests:
-`pytest aqueduct_tests/ -rP -vv`
+To run all SDK Integration Tests:
```
Comment (saurav-c) [inline integration_tests/sdk/README.md:34]: Does it matter which directory the the script is run from? I think it does based on the arg parsing below? If so, can you add instructions to this.

**Review by saurav-c — APPROVED** [review]
A couple nits, otherwise LGTM!

**Pull request merged**